### ISSUE TITLE
I'm trying to run this software in asahi fedora remix on my macbook m1 air. it's not running 

### ISSUE THREAD

**Issue by Ashir-Abrar** [OpenBangla/OpenBangla-Keyboard#357]
could you make this usable under asahi?

**Comment by larina3315** [comment]
I think you would have to compile it from source.

**Comment by Ashir-Abrar** [comment]
I'm new to linux and using fedora and kde desktop. please guide me how to do that

**Comment by Ashir-Abrar** [comment]
besides the latest support for fedora for this app is fedora38 but I am running fedora 39. would it still work?

**Comment by mominul** [comment]
You have to compile  OpenBangla Keyboard by following these commands:
```
sudo dnf -y update
sudo dnf -y install @buildsys-build rust cargo cmake qt5-qtdeclarative-devel libzstd-devel fcitx5 fcitx5-devel ninja-build
git clone --recursive https://github.com/OpenBangla/OpenBangla-Keyboard.git
cd OpenBangla-Keyboard
git checkout develop
git submodule update
mkdir build
cmake -Bbuild -DENABLE_FCITX=ON
ninja -C build
sudo ninja install -C build
```

After that, you have to follow the instructions from **step 2** of [this page](https://github.com/OpenBangla/OpenBangla-Keyboard/wiki/Configure-on-KDE-(Fcitx))

**Comment by larina3315** [comment]
@Ashir-Abrar I must also warn you, in Fedora KDE 39 on Wayland with KDE Plasma 5.27.9 I recently encountered a bug where despite installing fcitx and openbangla properly, Fcitx won't show up under the "Virtual Keyboards" section of KDE Settings Keyboard menu, preventing me from using OBK, this has forced me to return back to GNOME, although I donot know if it has been fixed after that or not.

I had it working perfectly fine on KDE Plasma 5.27.8 but the .9 point release broke it for me, it is very much unlikely to be an OBK issue and more like a KDE issue.

Also kinda offtopic but I recommend you stick with GNOME for sometime as a newcomer to Linux, as KDE is notorious for being buggy and glitchy.
